scripts/zkai_d64_commitment_consistency_method_probe.py

```
@@ -0,0 +1,562 @@
+        raise CommitmentConsistencyProbeError("generated_at must be a UTC timestamp string")
+    try:
+        parsed = dt.datetime.fromisoformat(value.removesuffix("Z") + "+00:00")
+    except ValueError as err:
+        raise CommitmentConsistencyProbeError("generated_at must be a valid UTC timestamp string") from err
+    if parsed.tzinfo != dt.timezone.utc:
+        raise CommitmentConsistencyProbeError("generated_at must be a UTC timestamp string")
+
+
+def _git_commit() -> str:
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)